provider/resources.go

```diff
@@ -232,6 +234,17 @@ func Provider() tfbridge.ProviderInfo {
 		},
 	}
 
+	err := tfbridgeX.ComputeDefaults(&prov, DefaultStrategy{
+		Resource: TokensKnownModules("newrelic_", "index", []string{
+			"alert", "obfuscation", "one", "infra", "nrql", "entity",
+			"eventsTo", "alertMuting", "logParsing", "apiAccess",
+			"service", "notifcation", "cloud", "synthetics", "obfuscation",
+		}, func(module, name string) (string, error) {
+			return fmt.Sprintf("cs101:%s:%s", module, name), nil
+		}).Resource,
+	})
```

Comment:
I suspect this was copy-pasted from the example with the `cs101` package, since it will form tokens prefixed with `cs101`.
```suggestion
	err := tfbridgeX.ComputeDefaults(&prov, TokensKnownModules("newrelic_", "index", []string{
			"alert", "obfuscation", "one", "infra", "nrql", "entity",
			"eventsTo", "alertMuting", "logParsing", "apiAccess",
			"service", "notifcation", "cloud", "synthetics", "obfuscation",
		}, tfbridgeX.MakeStandardToken(mainPkg)))
```

If you want this to apply to both datasources and resources, you don't need to create a `DefaultStrategy`, extract the `.Resource` field, and put it into another `DefaultStrategy`.